cmd/cloudshell_open/main.go

```diff
@@ -308,6 +309,20 @@ func run(opts runOpts) error {
 		}
 	}
 
+	hookEnvs := append(envs,
+		fmt.Sprintf("PROJECT_ID=%s", project),
+		fmt.Sprintf("SERVICE=%s", serviceName),
+		fmt.Sprintf("REGION=%s", region))
+
```

Comment:
let's also document that we add these.

Reply:
hmm I wonder if we should consider making K_SERVICE instead of SERVICE, that's what https://cloud.google.com/run/docs/reference/container-contract does.